examples/analytics-resource-timing.html

```diff
@@ -0,0 +1,50 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>AMP Analytics</title>
+  <link rel="canonical" href="analytics.amp.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+</head>
+<body>
+<p>Hi there!</p>
+<div id="container">
+<amp-analytics id="analytics1">
+<script type="application/json">
+{
+  "transport": {"beacon": false, "xhrpost": false},
+  "requests": {
+    "endpoint": "https://raw.githubusercontent.com/ampproject/amphtml/master/examples/img/ampicon.png"
+  },
+  "triggers": {
+    "defaultPageview": {
+      "on": "ini-load",
+      "request": "endpoint",
+      "extraUrlParams": {
+        "rt": "${resourceTiming}"
+      },
+      "resourceTimingSpec": {
+        "resources": {
+          "analytics": {
+            "host": "^(localhost:\\d+|cdn.ampproject.org)$",
+            "path": "v0/amp-analytics-0.1.*"
+          }
+        },
+        "encoding": {
+          "entry": "${key}.${initiatorType|substr:0:3}.${startTime}.${duration}",
```

Comment:
want to let you know we don't support filter function like ${initiatorType|substr:0:3} right now.

Reply:
I noticed that but forgot to update the example